src/auro-combobox.js

```diff
@@ -120,40 +126,48 @@ class AuroCombobox extends LitElement {
    * @returns {void}
    */
   handleMenuOptions() {
+    this.generateOptionsArray();
     this.availableOptions = [];
 
-    let noMatchOption = undefined; // eslint-disable-line no-undef-init
+    if (this.dynamicMenu) {
+      this.options.forEach((option) => {
+        this.availableOptions.push(option);
+      });
+    } else {
 
-    this.options.forEach((option) => {
-      let matchString = option.innerText.toLowerCase();
+      let noMatchOption = undefined; // eslint-disable-line no-undef-init
```

Comment:
nit-pick... is it better to inline a disable or add this to the .eslintrc?

Reply:
I think we should disable it. But let's address that as an update to our linter rules in the generator and push out consistent changes to components rather than make one off changes.

Reply:
I don't think we need to conflate this to a global thing. It's more of a question of how to best handle one-offs. In previous reviews, the feedback had gone both ways.